recipients_app/android/app/src/stage/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="org.socialincome.app">
```

Comment:
package='org.socialincome.app' is replace by the namespace 'org.socialincome.app' in build.gradle